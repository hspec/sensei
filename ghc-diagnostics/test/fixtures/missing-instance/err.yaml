version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/missing-instance/Foo.hs
  start:
    line: 3
    column: 12
  end:
    line: 3
    column: 14
severity: Error
code: 39999
message:
- |-
  No instance for `Eq (IO ())'
    arising from the first field of `Bar' (type `IO ()')
  Possible fix:
    use a standalone 'deriving instance' declaration,
      so you can specify the instance context yourself
- When deriving the instance for (Eq Bar)
hints: []
