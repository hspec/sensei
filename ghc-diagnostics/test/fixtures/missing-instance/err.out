test/fixtures/missing-instance/Foo.hs:3:12: error: [GHC-39999]
    • No instance for `Eq (IO ())'
        arising from the first field of `Bar' (type `IO ()')
      Possible fix:
        use a standalone 'deriving instance' declaration,
          so you can specify the instance context yourself
    • When deriving the instance for (Eq Bar)

